src/webapp/azext_webapp/_help.py

```diff
@@ -59,3 +59,8 @@
         - name: Upload files from the current directory to an Azure Container Registry, then build a container image and deploy it to a web app. The Azure Container Registry must already exist.
           text: az webapp container up -n AppName --registry-rg ContainerRegistryResourceGroup --registry-name ContainerRegistryName
 """
+
+helps['webapp deploy'] = """
+    type: command
+    short-summary: Deploys a given artifact to Azure Web Apps through OneDeploy.
```

Comment:
isn't this going to be scoped only for certain extensions currently? Like .war, .zip so would be good to be clear. Please check these help content with your PM as well

Reply:
The OneDeploy API can deploy any extension to a webapp as a static file. However it does take specific actions for certain extensions such as .war, .jar, and .zip.